_includes/nav/page-item.html

```diff
@@ -1,8 +1,8 @@
 {% include nav/current.html page=include.page %}
 <a href="{{ include.page.url | relative_url }}"{%
-  if _current -%} class="usa-current"{% endif %}>{{
-    include.page.title | smartify
-  }}</a>
+  if _current -%} class="usa-current"{% endif %}>
+  <span>{{ include.page.title | smartify }}</span>
```

Comment:
_Re-adding in this new PR + moving to the correct line of code_ 
@donjo do you know how I can add the `<span>` to the header nav but not the sidenav?

Reply:
Ok in trying to figure out how all of this is sew together and I think I now understand. Which part are you trying to ensure doesn't happen on the sidenav? just the "current" styling that adds the blue bar on the left or are you trying to omit something from the sidenav?

Reply:
I just don't want the `<span>` wrapped around the title in the sidenav but do want it in the main nav. Lmk if that makes sense.

Header nav: `<a href=""><span>Title</span></a>`
Sidenav: `<a href="">Title></a>`

There are actually no visual changes from this, it's just the markup.

Reply:
Ok that makes sense. 

🤔  _thinking_

Reply:
not sure if there is a more elegant way to write this but this works

```
<a href="{{ include.page.url | relative_url }}"{%
  if _current -%} class="usa-current"{% endif %}>
  {% unless sidenav %}<span>{% endunless %}
  {{ include.page.title | smartify }}
  {% unless sidenav %}</span>{% endunless %}
</a>
```

Reply:
Perfect 🎉 Added in 3942ae6.